src/prefect/cli/cloud.py

```diff
@@ -426,7 +429,7 @@ async def login(
         )
 
         if choice == "key":
-            key = typer.prompt("Paste your authentication key")
+            key = typer.prompt("Paste your authentication key", hide_input=True)
```

Comment:
is there any additional risk of this data being saved locally? shell history, etc?

i think the answer is no, but want to confirm

Reply:
Yeah I don't think so. This matches our previous behavior.